sirepo/template/ml.py

```diff
@@ -736,6 +744,43 @@ def _get_fit_report(report, x_vals, y_vals):
     return param_vals, param_sigmas, plots
 
 
+def _get_remote_data(url, headers_only):
+    import os
```

Comment:
Move these imports up top and fully qualify them.

Reply:
[62cb8c7](https://github.com/radiasoft/sirepo/pull/4671/commits/62cb8c7be3a8ee06ce5e89ed8e3ce2ea1e7e708c)